### PR TITLE
GH-1117 Cypress homepage e2e tests

### PR DESCRIPTION
**Pull request by hackergil**
Closes GH-1117

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/integration/homepage-e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    cy.get('picture.ElectronicsHompageSplashBannerComponent').should('exist');
+  });
+
+  it('should have footer with footer navigation and notice', () => {
+    const footer = cy.get('cx-dynamic-slot.Footer');
+    footer.should('exist');
+    footer.within(() => {
+      cy.get('.navigation-elements').should('have.length', 3);
+      cy.get('h1').should('have.length', 3);
+      cy.get('cx-generic-link').should('exist');
```
Comment (LTiger14) [inline projects/storefrontapp-e2e-cypress/cypress/integration/homepage-e2e-spec.ts]: Is this line to test that you have at least one link?

Comment (LTiger14) [inline projects/storefrontapp-e2e-cypress/cypress/integration/homepage-e2e-spec.ts]: Is this line to test that you have at least one link?

Comment (hackergil) [reply projects/storefrontapp-e2e-cypress/cypress/integration/homepage-e2e-spec.ts]: Yes. 3 titles, at least one link

Comment (LTiger14) [reply projects/storefrontapp-e2e-cypress/cypress/integration/homepage-e2e-spec.ts]: Sounds good

**Pull request merged**